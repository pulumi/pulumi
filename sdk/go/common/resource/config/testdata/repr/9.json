{"my:mapKey":{"a":"b","c":"d"}}