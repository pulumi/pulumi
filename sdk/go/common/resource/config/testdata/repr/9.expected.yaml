decrypt:
  my:mapKey: '{"a":"b","c":"d"}'
paths:
  my:mapKey:
    value:
      a: b
      c: d
    string: '{"a":"b","c":"d"}'
    redacted: '{"a":"b","c":"d"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAh/4IeAAIBYQZzdHJpbmcMAwABYgFjBnN0cmluZwwDAAFk
    secure: false
    isObject: true
  my:mapKey.a:
    value: b
    string: b
    redacted: b
    object: DhAABnN0cmluZwwDAAFi
    secure: false
    isObject: false
  my:mapKey.c:
    value: d
    string: d
    redacted: d
    object: DhAABnN0cmluZwwDAAFk
    secure: false
    isObject: false
